eu.ad5001.LogarithmPlotter.json

```diff
@@ -0,0 +1,43 @@
+{
+    "app-id": "eu.ad5001.LogarithmPlotter",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "finish-args": [
+        "--share=ipc",
+        "--device=dri",
+        "--socket=x11",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.DBus.Proprieties",
+        "--talk-name=org.freedesktop.IBus",
+        "--talk-name=org.freedestkop.portal.*",
```

Comment:
What is it needed for?

Reply:
I'm slightly confused as to why you need to talk to the portal for open/save but also have the --filesystem=home permission.